test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/RestoreCommandTests.cs

```diff
@@ -1250,20 +1250,20 @@ public async Task Restore_WithHttpSourceAndFalseAllowInsecureConnections_WarnsCo
             CommandRunnerResult result = RunRestore(pathContext, _successExitCode);
 
             // Assert
-            string formatString = "You are running the 'restore' operation with an 'HTTP' source, '{0}'. Non-HTTPS access will be removed in a future version. Consider migrating to an 'HTTPS'";
-            string warningForHttpSource = string.Format(formatString, "http://api.source/index.json");
-            string warningForHttpsSource = string.Format(formatString, "https://api.source/index.json");
+            string errorForHttpSource = string.Format(PackageManagement.Strings.Error_HttpSource_Single, "restore", "http://api.source/index.json");
+            string errorForHttpsSource = string.Format(PackageManagement.Strings.Error_HttpSource_Single, "restore", "https://api.source/index.json");
 
             result.Success.Should().BeTrue();
```

Comment:
If AllowInsecureConnections is false, how can Success be true?

Reply:
You are right. Previously it was succeeding because restore would log the error and still proceed restoring. No I have made sure it returns after logging the error.